Lib/test/test_tkinter/test_misc.py

```diff
@@ -4,7 +4,7 @@
 from tkinter import TclError
 import enum
 from test import support
-from test.test_tkinter.support import AbstractTkTest, AbstractDefaultRootTest, requires_tk
+from test.test_tkinter.support import AbstractTkTest, AbstractDefaultRootTest, requires_tk, tk_version
```

Comment:
Nit: dangling unused import:

```suggestion
from test.test_tkinter.support import AbstractTkTest, AbstractDefaultRootTest, requires_tk
```

Reply:
I pushed one more commit containing these changes and the ones above.